### DISCUSSION TITLE
Remove the bottom border of the paginator inside the Datatable component

### DISCUSSION THREAD

**Discussion by Lee-Margo** [primefaces/community#2623] [PrimeVue] [answered]
Hello there, 

I am having difficulty finding the right way to remove the bottom border of the paginator that comes with the DataTable component. It would be great if someone could tell me what I should do to remove it, whether using Tailwind CSS or standard CSS styles.

![image](https://github.com/user-attachments/assets/13a2205e-6b78-4c59-9ad8-9447b50e6bba)
![image](https://github.com/user-attachments/assets/8311dbd8-219a-43a2-a8d3-16b1ad7a2391)

**Comment by TheOredor** [comment, marked as answer]
Hello, 

I'm have the same problem, but i have a simple solition.

In the Datatable Component add the prop style with this line:

`--p-datatable-paginator-bottom-border-width: 0`

```VUE
<DataTable
    :value="itemsFiltered"
    class="responsive-table"
    stripedRows
    :loading="loading"
    :filters="filters"
    filterDisplay="menu"
    paginator
    :rows="15"
    :rowsPerPageOptions="[15, 30, 50, 100]"
    paginatorTemplate="FirstPageLink PrevPageLink PageLinks NextPageLink LastPageLink CurrentPageReport RowsPerPageDropdown"
    currentPageReportTemplate="{first} al {last} de {totalRecords}"
    style="--p-datatable-paginator-bottom-border-width: 0"
>
```

Before:
<img width="926" height="104" alt="image" src="https://github.com/user-attachments/assets/28dcfa66-de43-4f11-83b8-d34b27c3902a" />

After:
<img width="931" height="82" alt="image" src="https://github.com/user-attachments/assets/dd916a81-6b02-4cd1-9c1a-b5291aea1cf8" />

**Reply by Lee-Margo** [reply]
thanks a lot, will definitely try in my latest project :)
